collection/shubham.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name" : "shubham",
+	"meaning" : "auspicious, lucky",
+	"aliases" : ["shubh"],
+	"translations" : {
+		"jap" : "シュハム",
```

Comment:
Can you please update this to use the correct ISO code? The correct ISO code for Japanese is `jpn`.